lib/matplotlib/axes/_base.py

```diff
@@ -3349,6 +3347,11 @@ def _set_view_from_bbox(self, bbox, original_view, direction='in',
         Xmin, Xmax = self.get_xlim()
         Ymin, Ymax = self.get_ylim()
 
+        if(original_view is not None):
+            x0, x1, y0, y1 = original_view
+        else:
+            x0, x1, y0, y1 = Xmin, Xmax, Ymin, Ymax
+
```

Comment:
I understand but I think this little change deserves another PR. Unless I'm mistaken this is not related to the `Zoom` problem